assets/base.css

```diff
@@ -1122,11 +1123,11 @@ summary::-webkit-details-marker {
   }
 
   .slider--tablet.grid--peek .grid__item:first-of-type {
-    padding-left: 1.5rem;
+    margin-left: 1.5rem;
   }
 
   .slider--tablet.grid--peek .grid__item:last-of-type {
-    padding-right: 1.5rem;
+    margin-right: 1.5rem;
   }
 }
 
```

Comment:
Noticed that the counter will change it's width as you go through it when you go from single digit number to two digit: [video](https://screenshot.click/04-44-8ybpv-mgvi1.mp4)

Not related to the issue this PR is trying to solve but could be a quick easy win added to the PR description 😬

Reply:
See any problem with this approach? https://github.com/Shopify/dawn/pull/1297/commits/c72df71dd41eb775c60432db73ee6d2c2cedd9e6

Reply:
Hmm I wonder about font scale and different font families: [video](https://screenshot.click/07-59-vehqb-4lp1s.mp4)

Reply:
Seemed relatively safe to me, but the arrows could get closer of course. I don't have real strong feelings on it, the width changing at double digits didn't even bother me that much. If we want to consider it more or implement a more advanced solution I'd rather open an issue for it. Happy to revert this change or not.

Reply:
Yeah true. No one likes that. Will leave as is for now.